README.md

```diff
@@ -31,37 +27,10 @@ cd dkls
 wasm-pack build --release --target web
 ```
 
-#### Step 2
+### Step 2
 
-From the `pkg` subfolder in `dkls`:
+Copy the files from `dkls/pkg` (except the WASM and the package.json) to `packages/tss-lib`.
```

Comment:
This isn't reflected in the PR, all the files are currently under pkg. Could we not instead use pkg as-is?

Reply:
updated the readme